spec/validator.spec.ts

```diff
@@ -16,7 +17,7 @@ import { ResearchStudy } from '../src/research-study';
  * being made properly.
  */
 import { exec } from 'child_process';
-import * as fs from 'fs';
+import fs from 'fs';
```

Comment:
This should probably be updated with the new validation work, once it is ready.

Reply:
This has been done.